backend/backendutil/envelope.go

```diff
@@ -5,11 +5,16 @@ import (
 	"strings"
 
 	"github.com/emersion/go-imap"
+	"github.com/emersion/go-message/charset"
```

Comment:
I'd like to make the dependency to `go-message/charset` optional. Fortunately we already have `imap.CharsetReader` for this purpose.